chapters/interface.tex

```diff
@@ -47,7 +47,7 @@ \section{Interface Terminology}\label{interface-terminology}
 \lstinline!A! is a restricted subtype of \lstinline!B!, or equivalently, the interface of \lstinline!A! is plug compatible to the interface of \lstinline!B!, if \lstinline!A! is a subtype of
 \lstinline!B! and if connector components in \lstinline!A! that are not in \lstinline!B!, are default connectable.
 \begin{nonnormative}
-E.g.\ it is not allowed that these connectors have variables with the \lstinline!input! prefix, because then they must be connected.
+e.g., it is not allowed that these connectors have variables with the \lstinline!input! prefix, because then they must be connected.
```

Comment:
```suggestion
E.g., it is not allowed that these connectors have variables with the \lstinline!input! prefix, because then they must be connected.
```